config/deploy.rb

```diff
@@ -229,8 +229,11 @@
       abort "Exiting because you didn't type 'yes'" unless STDIN.gets.chomp == 'yes'
     end
 
-    backup.db.dump
-    backup.db.trim
+    print "Backup database? [no/YES] ".colorize(:green)
+    unless STDIN.gets.chomp.downcase == 'no'
```

Comment:
It used to always backup the DB, but we added this option because backing up the DB after huge amounts of Trove was ingested took a long time so we added this option. An accidental backup is going to be pretty inconsequential.